test.txt

```diff
@@ -0,0 +1 @@
+ytestt
```

Comment:
Could you remove the 'y' and the second 't' at the end please?

Reply:
done